openedx/core/djangoapps/user_api/serializers.py

```diff
@@ -22,7 +22,11 @@ def get_name(self, user):
         """
         Return the name attribute from the user profile object
```

Comment:
@ahsan-ul-haq We need to update the doc string here, as now this method is returning `None` in case user profile does not exist.